spec/support/helpers/ask_export_helper.rb

```diff
@@ -22,71 +22,59 @@ def smart_survey_response(items)
   def smart_survey_row(options = {})
     environment = options.fetch(:environment, :draft)
     config = AskExport::CONFIG[environment]
+
+    to_merge = {}
+    if options[:client_id]
+      to_merge[:variables] = [{ id: random_id,
+                                name: "_ga",
+                                label: "clientID",
+                                value: options[:client_id] }]
+    end
+
+    pages = [
```

Comment:
I found this a little confusing, initially, until I got to [L49](https://github.com/alphagov/govuk-ask-export/pull/8/commits/6936d59f96e9f2c15ff0b9b48f6a241042cb8efb#diff-ebfbdbe7e776cd6959b8b8bbc07f3c7cR49). Could we have a `smart_survey_age_check_page`, for consistency?